public/app/features/admin/UpgradePage.tsx

```diff
@@ -122,16 +122,20 @@ const ServiceInfo = () => {
           title={t('admin.service-info.title-unlimited-expert-support', 'Unlimited Expert Support')}
           image="public/img/licensing/customer_support.svg"
         >
-          24 x 7 x 365 support via
+          <Trans i18nKey="admin.service-info.year-round-support">24 x 7 x 365 support via</Trans>
```

Comment:
really minor - but maybe can you change this to a `×` / `&times;` symbol please?